elements/mpich-3.3.1/post-install.d/05-mpich

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+if [ ${DIB_DEBUG_TRACE:-0} -gt 0 ]; then
+    set -x
+fi
+set -o errexit
+set -o nounset
+set -o pipefail
+
+#############################################################
+# Pre-Installation
+#############################################################
+
+MPICH_TMP_FOLDER=/root/tmp
+mkdir -p $MPICH_TMP_FOLDER
+
+#############################################################
+# Installation (runfile)
+#############################################################
+
+cd $MPICH_TMP_FOLDER
+
+# Get the MPICH 3.3.1
+wget -nv https://www.mpich.org/static/downloads/3.3.1/mpich-3.3.1.tar.gz
+tar xvfz mpich-3.3.1.tar.gz && cd mpich-3.3.1
+
+./configure
+
+#############################################################
+# Cleaning
+#############################################################
+
+# Clean the tmp folder used by CUDA
+rm -rf $MPICH_TMP_FOLDER
```

Comment:
Can we keep some examples, like hello_world, somewhere before deleting the installation directory?

Reply:
I added a step to copy the `cpi` example that is built with MPICH in `/usr/local/bin`. It's also possible to keep the whole `examples` directory from within the MPICH source package if we want.

Reply:
One example should be good enough. We'll see if users ask more in the future. Thank you for the quick fixes, and I'll try deploy again!